core/string/translation.cpp

```diff
@@ -1316,6 +1618,11 @@ void TranslationServer::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("clear"), &TranslationServer::clear);
 
 	ClassDB::bind_method(D_METHOD("get_loaded_locales"), &TranslationServer::get_loaded_locales);
+
+	ClassDB::bind_method(D_METHOD("is_pseudolocalization_enabled"), &TranslationServer::is_pseudolocalization_enabled);
+	ClassDB::bind_method(D_METHOD("set_pseudolocalization_enabled", "enabled"), &TranslationServer::set_pseudolocalization_enabled);
```

Comment:
You can add a `pseudolocalization_enabled` property with `ADD_PROPERTY` additionally to the setter and getter. This will remove the setter and getter from the docs and expose the property instead, which is more user-friendly.

For its documentation, you can copy what you had in the setter, and edit it to start following our style for boolean property descriptions: `If [code]true[/code], ...`. (No need to say what happens if false since nothing happens.)